### PR TITLE
[feat] Navigation w/Hamburger menu

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully added mobile navigation menu to navigation component.
This allows user to have a basic navigation menu setup that can be used easily and is responsive for all screen sizes

Features:
1. Hamburger menu onclick opens sidemenu with various button/options
2. User can easily close sidemenu with click of close button

## Deployment

https://deploy-preview-45--powertrain-ui-temp.netlify.app/documentation/html/navigation

### CODE REVIEW

File: Documentation/js/mobile-nav.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const hamburger_mobile = document.getElementsByClassName("hamburger")[1];
+const close_sidemenu = document.getElementsByClassName(
+  "mobile-sidebar-close-btn"
+)[0];
+const sidemenu = document.getElementsByClassName("mobile-sidebar")[0];
+
+hamburger_mobile.addEventListener("click", () => {
+  console.log("click");
```
Comment (junaidshaikh-js) [inline Documentation/js/mobile-nav.js]: remove console logs

File: Documentation/js/mobile-nav.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  sidemenu.style.width = "50%";
+  sidemenu.style.overflow = "auto"
+});
+
+close_sidemenu.addEventListener("click", () => {
+  console.log("click");
```
Comment (junaidshaikh-js) [inline Documentation/js/mobile-nav.js]: Here too

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Clean code, can be merged

**Review by lannisterrr — APPROVED** [review]
good use of transition and animation. code seems good to merge.

File: Documentation/js/mobile-nav.js (hunk 1 above)
Comment (Hamdrive) [reply Documentation/js/mobile-nav.js]: resolved!

File: Documentation/js/mobile-nav.js (hunk 2 above)
Comment (Hamdrive) [reply Documentation/js/mobile-nav.js]: resolved! thanks for catching it

**Comment by Hamdrive** [comment]
Successfully got PR reviewed by 2 teammates, fixes were suggested and implemented, and deployment is functional.
Thus this PR can be closed and merged to temp-dev

**Pull request merged**